templates/navigation/menu.html.twig

```diff
@@ -25,7 +25,7 @@
         %}
         <ul{{ attributes.addClass(menu_classes) }}>
       {% else %}
-        <ul class="nav__subnav">
```

Comment:
The Sass and JS were both looking for nav--subnav, so seemed easiest to just change this one thing to match everything else to get it working again, but doesn't matter to me. If going the other direction, just have to track down all the places it needs to change.